docs/installation/air-gapped.md

```diff
@@ -83,3 +83,18 @@ For example, to install with NVIDIA and start the GPUStack server **with the bui
 ```
 
 If your accelerator is not NVIDIA, adjust the startup command accordingly.
+
+### Pulling Inference Backend Images from a Secure Registry
+
+If your internal container registry requires authentication,  
+set the following environment variables when starting the GPUStack worker to allow it to pull the runner image.
+
+```diff
+ sudo docker run -d --name gpustack \
+     ...
++    --env GPUSTACK_RUNTIME_DEPLOY_DEFAULT_REGISTRY_USERNAME=<your_internal_registry_username> \
```

Comment:
how do we explain the `DEFAULT` here, is there a way to set non-default ones?

Reply:
I have no idea that how to configure a non-default registry in gpustack, the `DEFAULT_REGISTRY` explain in here, see https://github.com/gpustack/runtime/blob/572927555257f7ee1ef8c99cae9d6d4cf526fbc2/gpustack_runtime/envs.py#L61-L74. And we also have an option named `--system-default-container-registry`, so I think they are the same.